hazelcast-jet-reference-manual/src/main/asciidoc/work-with-jet/configure-fault-tolerance.adoc

```diff
@@ -239,3 +239,40 @@ members must be present after the restart for Jet to be able to reload
 the data. Beyond that, there's no special action to take: as soon as the
 cluster re-forms, it will automatically reload the persisted snapshots
 and resume the jobs.
+
+= Configure Blue/Green Client (Enterprise Only)
+
+Blue/Green deployment refers to a software technique which can be used
+to reduces system downtime by deploying two mirrored production
+environments referred to as Blue and Green. One of the environments is
+running in production while the other one is active standby.
+
+The Jet client can be configured to switch to the other cluster in case
+of a connection failure on production environment.
+
+== Programmatic Configuration
+
+[source]
+----
+include::{javasource}/ConfigureFaultTolerance.java[tag=s7]
+----
+
+== Declarative Configuration
+
+[source]
+----
+<hazelcast-client-failover>
+    <try-count>10</try-count>
+    <clients>
+        <client>hazelcast-client-A.xml</client>
+        <client>hazelcast-client-B.xml</client>
+    </clients>
+</hazelcast-client-failover>
+----
+
+After switching to the cluster-B, any running job on cluster-A will be
```

Comment:
`After switching to the cluster-B, any running job on cluster-A will be canceled` - this is not correct - connection to cluster-A is lost from client but it does not mean that running jobs on cluster-A will be canceled. 

Moreover it can be problematic for users - original job can be still running in cluster-A then resubmit them to another cluster can cause unexpected data duplication in remote sink. Maybe we should specify that Blue/Green in Jet context is mostly applicable for short living batch jobs (i.e. queries expressed as Jet Jobs).